Other/MountDoomSkull.cs

```diff
@@ -0,0 +1,54 @@
+//cs_include Scripts/CoreBots.cs
```

Comment:
should be in the story > chaos folder but if it works then great

Reply:
Yes it works

Reply:
Yep it’s standalone

Reply:
ight ill move it after then